src/components/learner-credit-management/members-tab/GroupMembersCsvDownloadTableAction.jsx

```diff
@@ -69,6 +70,13 @@ const GroupMembersCsvDownloadTableAction = ({
     });
   };
 
+  let buttonSelectedNumber = 0;
+  if (selectedEmailCount > 0) {
+    buttonSelectedNumber = isEntireTableSelected ? tableInstance.itemCount : selectedEmailCount;
```

Comment:
[issue] small missing detail but there should be parenthesis around the selected number of learners in the download button e.g. `Download (12)`

perhaps something like this and update the unit test as such:

```
buttonSelectedNumber = isEntireTableSelected ? `(${tableInstance.itemCount})` : `(${selectedEmailCount})`;
```